requirements.txt

```diff
@@ -6,4 +6,5 @@ pytest==7.4.0
 transformers==4.35.0
 diffusers==0.21.4
 torch==2.1.0
-stability-sdk==0.8.4
\ No newline at end of file
+stability-sdk==0.8.4
+pydub==0.25.1
```

Comment:
Can we also update dependencies in `pyproject.toml`?

Reply:
Added